test/k8sT/DatapathConfiguration.go

```diff
@@ -580,14 +580,18 @@ var _ = Describe("K8sDatapathConfig", func() {
 		})
 	})
 
-	Context("ManagedEtcd", func() {
-		AfterAll(func() {
-			deleteETCDOperator(kubectl)
-		})
-		It("Check connectivity with managed etcd", func() {
+	Context("Etcd", func() {
+		It("Check connectivity", func() {
+			deploymentManager.Deploy(helpers.CiliumNamespace, StatelessEtcd)
+			deploymentManager.WaitUntilReady()
+
```

Comment:
It seems to me the etcd pod is never removed after this test?

Reply:
It is removed via `deploymentManager.DeleteAll()` in `AfterEach`. This ensures that everything that was deployed via the deployment manager is cleaned up.